pkg/gce-pd-csi-driver/utils_linux.go

```diff
@@ -47,7 +48,25 @@ func getDevicePath(ns *GCENodeServer, volumeID, partition string) (string, error
 	return devicePath, nil
 }
 
-func formatAndMount(source, target, fstype string, options []string, m *mount.SafeFormatAndMount) error {
+func (ns *GCENodeServer) formatAndMount(source, target, fstype string, options []string, m *mount.SafeFormatAndMount) error {
+	if ns.formatAndMountSemaphore != nil {
+		done := make(chan any)
+		defer close(done)
+
+		ns.formatAndMountSemaphore <- struct{}{}
```

Comment:
can we put a comment here this call blocks until the formatAndMountSemaphore channel becomes empty again (equivalent to an acquire())

Reply:
done